client/src/partials/users/users.routes.js

```diff
@@ -0,0 +1,59 @@
+
+angular.module('bhima.routes')
+  .config(['$stateProvider', function ($stateProvider) {
+
+    $stateProvider
+      .state('users', {
+        abstract : true,
+        url : '/users',
+        controller: 'UsersController as UsersCtrl',
+        templateUrl: 'partials/users/users.html'
+      })
+
+      .state('users.create', {
+        url : '/create',
+        params : {
+          creating : { value : true }
+        },
+        onEnter :['$uibModal', usersModal]
+      })
+      .state('users.list', {
+        url : '/:id',
+        params : {
+          id : { squash : true, value : null }
+        }
+      })
+      .state('users.edit', {
+        url : '/:id/edit',
+        params : {
```

Comment:
So, I just tested it with Billing Services, and my proposal will not work.  Don't work about it.